### DISCUSSION TITLE
[Esame Ottobre 2023] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#376] [Esami - Bandiera]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/fc194b5a-d7a4-49fe-a352-77920edd5a7a)

**Comment by Elia-Belli** [comment]
**(a)** Calcoliamo prima le permutazioni inverse:

$$\sigma_1^{-1}=\begin{pmatrix}1&2&3&4&5&6&7&8&9\\
7&6&5&8&1&9&3&4&2\end{pmatrix}\ ;\ \sigma_2^{-1}=\begin{pmatrix}1&2&3&4&5&6&7&8&9\\
6&9&1&7&5&3&8&2&4\end{pmatrix}\ ;\ \sigma_3^{-1}=\begin{pmatrix}1&2&3&4&5&6&7&8&9\\
7&1&5&8&3&2&6&9&4\end{pmatrix}$$

$$\sigma_1^{-1}\sigma_2^{-1}\sigma_3^{-1}=\begin{pmatrix}1&2&3&4&5&6&7&8&9\\
4&9&1&6&7&2&5&8&3\end{pmatrix}$$

**(b)** Decomponiamo le permutazioni in cicli:

$$\sigma_1=(1\ 5\ 3\ 7)(2\ 9\ 6)(4\ 8)\Rightarrow \sigma_1\ è\ pari$$

$$\sigma_2=(1\ 3\ 6)(2\ 8\ 7\ 4\ 9)\Rightarrow \sigma_2\ è\ pari$$

$$\sigma_3=(1\ 2\ 6\ 7)(3\ 5)(4\ 9\ 8)\Rightarrow \sigma_3\ è\ pari$$

L'ordine di una permutazione è dato dall'mcm degli ordini dei cicli in cui si decompone, sappiamo che un k-ciclo ha ordine k:
$$o(\sigma_1)=mcm(4,3,2)=12$$

$$o(\sigma_2)=mcm(3,5)=15$$

$$o(\sigma_3)=mcm(4,2,3)=12$$

**(c)** Solamente $\sigma_1$ e $\sigma_3$ hanno stessa struttura ciclica, allora sono le uniche permutazioni coniugate; troviamo $\alpha$ tale che $\sigma_3=\alpha \sigma_1 \alpha^{-1}$ che in tal caso è unica:

$$\alpha=\begin{pmatrix}1&2&3&4&5&6&7&8&9\\
1&4&6&3&2&8&7&5&9\end{pmatrix}$$

**Reply by Alby7503** [reply]
Non dovrebbero la prima e la terza essere dispare dato che si possono scrivere con un numero dispari di trasposizioni?

**Reply by Elia-Belli** [reply]
no si decompongono in numero pari di trasposizioni, ad esempio $\sigma_1=(1\ 7)(1\ 3)(1\ 5)(2\ 6)(2\ 9)(4\ 8)$ se decomposto in trasposizioni

**Comment by Frisbon** [comment]
Analogo ad Elia
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/f056d382-8411-4b29-8d5e-67e673ded16d)
